tests/Flunt.Br.Tests/PhoneContractTest.cs

```diff
@@ -50,6 +50,26 @@ public class PhoneContractTest
         [DataRow("+5523 232399999999999")]
         [DataRow("+55 99 999923 923999")]
         [DataRow("+55 99 99999 992399")]
+        [DataRow("(99) 9999-9999")]
```

Comment:
@luizgamabh estou um pouco preocupado em relação a considerar esse e alguns outros formatos como inválidos.

Reply:
@lira92 eu considerei inválido por ser um número de celular e estar sem o dígito 9 que já foi homologado em todo o Brasil... Vai passar por exemplo:
(99) 4004-9999
(99) 1999-9999
(99) 2999-9999
(99) 3999-9999
(99) 4999-9999
(99) 5999-9999
(99) 6999-9999
(99) 7999-9999

Não vai passar
(99) 8999-9999
(99) 9999-9999

Vai passar
(99) 9 8999-9999
(99) 9 9999-9999
(99) 9-8999-9999
(99) 9-9999-9999
etc...

Eu segui as orientações do plano nacional de numeração da Anatel. Mas se preferir posso liberar os números iniciados com 8 e 9 sem o dígito 9 obrigatório.

Reply:
@lira92 desculpe pelos vários commits, mas não sou dev C#, não consegui rodar local os testes, instalei tudo que o VSCode sugeriu, mas não consegui, enfim, ideal fazer um squash se for aceitar o PR... Só estou tentando contribuir ;)

Para o momento mantive retrocompatibilidade com os testes que você montou inicialmente, adicionei dois parâmetros OPCIONAIS no Validate
bool strictNineDigit -> obriga usar o dígito 9 em números de celular, por padrão deixei false
bool cellPhonesOnly -> só valida caso o número seja um celular, por padrão false tb.
Podem ser usados combinados, ou não...

Aproveitando criei o método IsNewFormatCellPhone que vai validar somente números de celular com o nono dígito.

Espero ter ajudado.

Reply:
@luizgamabh perfeito, possivelmente farei alguns testes e alterações locais e lanço um novo release, muito obrigado pela sua contribuição!!